Cargo.toml

```diff
@@ -21,6 +21,20 @@ strum_macros = "0.25"
 radix_trie = "0.2.1"
 lazy_static = "1.4.0"
 regex = "1.9.5"
+clap = { version = "4.4", features = ["wrap_help", "suggestions"] }
+sysexits = { version = "0.7", features = ["std"] }
+build-time = "0.1"
+hcl-rs = "0.16"
+daemonize = "0.5"
+actix-web = { version = "4.4", features = ["openssl"] }
+actix-tls = "3.1"
+actix-rt = "2.9"
+log = "0.4"
+env_logger = "0.10"
+
+[[bin]]
+name = "rusty_vault"
```

Comment:
Is it possible to use rusty-vault (stick, instead of underscore) as the binary name? Or even just use some simple name such as 'rvault' or so...

Reply:
undersocres require more key types to input on the command line

Reply:
Has been modified to rvault.